docs/ref/files/storage.txt

```diff
@@ -11,7 +11,18 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    A dictionary-like object that allows you to retrieve a storage instance
+    using its alias as defined by :setting:`STORAGES`.
+
+    ``storages`` has a dictionary attribute ``backends``, which at startup is
```

Comment:
Since we already said `storages` is a dict-like (but not **necessarily** a dict), I would be less prescriptive here by saying "has a key" (which is really what the code cares about, no?).
Also, I'm not sure what "startup" is... is it on server first start? reload? all the above? How about something like "on initial settings load" or even the simpler (but more vague, so that's a con) "defaults"?
```suggestion
    ``storages`` has a key ``backends``, which defaults to
```

Reply:
It's not a key per se... 

You do storage lookup via dictionary (key) access `storages["compressor"]`  but access backends as `storages.backends`.

Reply:
Thanks for the clarification, then I guess we could say just "attribute".